src/unix/linux_like/linux/gnu/mod.rs

```diff
@@ -755,7 +755,15 @@ pub const RTLD_DI_TLS_MODID: c_int = 9;
 pub const RTLD_DI_TLS_DATA: c_int = 10;
 
 pub const SOCK_NONBLOCK: c_int = O_NONBLOCK;
+
 pub const PIDFD_NONBLOCK: c_uint = O_NONBLOCK as c_uint;
+// libc-test failure. To new?
+// pub const PIDFD_THREAD: c_uint = O_EXCL as c_uint;
+
+// libc-test failure. To new?
+// pub const PIDFD_SIGNAL_THREAD: c_uint = 1;
+// pub const PIDFD_SIGNAL_THREAD_GROUP: c_uint = 2;
+// pub const PIDFD_SIGNAL_PROCESS_GROUP: c_uint = 4;
```

Comment:
It looks like these were only added in the past year. You can update `libc-test/build.rs` to skip these constants in testing, they'll get re-enabled once we update to a kernel version that has them.